fe/fe-core/src/main/java/com/starrocks/analysis/MultiRangePartitionDesc.java

```diff
@@ -127,6 +130,13 @@ public void setTimeUnit(String timeUnit) {
         List<SingleRangePartitionDesc> singleRangePartitionDescs = Lists.newArrayList();
         long currentLoopNum = 0;
         long maxAllowedLimit = Config.max_partitions_in_one_batch;
+
+        int dayOfWeek = 1;
```

Comment:
1. Please give some comments for this logical.
2. How about logic of adding partition.